packages/@okta/vuepress-theme-prose/components/FrontPageCodeMirror.vue

```diff
@@ -31,13 +31,10 @@ export default {
   data() {
     return {
       widgetCode: `
-// Uses okta-signin-widget version -=OKTA_REPLACE_WITH_WIDGET_VERSION=-
```

Comment:
I'm not sure if we should remove. We might still want this text (and the other comments below from the code snippet) still